analyzer/src/main/kotlin/managers/Gradle.kt

```diff
@@ -186,7 +186,7 @@ class Gradle(
         val projectDir = definitionFile.parentFile
         val gradleConnection = gradleConnector.forProjectDirectory(projectDir).connect()
 
-        return temporaryProperties(*gradleSystemProperties.toTypedArray()) {
+        return temporaryProperties("user.dir" to projectDir.absolutePath, *gradleSystemProperties.toTypedArray()) {
```

Comment:
I'm not sure, I'd need to check. Is this your indirect way of suggesting that this property should only be set to the *root* project directory, but not to the directory of the respective subprojects?

Reply:
> Is this your indirect way of suggesting that this property should only be set to the root project directory, but not to the directory of the respective subprojects?

It wasn't meant as suggestion. I wouldn't know right away what to suggest. So I asked in the hope that the answer is such that there is no need to think about that case any further. (I was afraid that it might break something)

Reply:
> this property should only be set to the root project directory

But actually this is indeed the case, so thank for making me think about that. I've pushed a new iteration that addressed this.